ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -3911,26 +3941,35 @@ tests:
       E2E_RUN_TAGS: '@amd64 and @vsphere-ipi and @proxy and @network-ovnkubernetes
         and not @fips'
       TAG_VERSION: '@4.14'
+    leases:
+    - env: VSPHERE_CONNECTED_LEASED_RESOURCE
+      resource_type: vsphere-connected-2-quota-slice
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-vsphere-ipi-proxy-workers-rhel8
 - as: vsphere-ipi-remote-worker-f28
   cron: 1 19 8 * *
   steps:
-    cluster_profile: vsphere-2
+    cluster_profile: vsphere-connected-2
```

Comment:
I guess this one is not for disconnected env install.

Reply:
yes, it's for remote worker on connected env. remote worker also use hardcode, so I changed it too.

Reply:
I think current remote worker job would fail if use cluster_profile: vsphere-2

Reply:
Okay, ignore my comments, I think I got the reason.